packages/p2p/src/components/modal-manager/modals/my-profile-balance-modal/my-profile-balance-modal.tsx

```diff
@@ -12,10 +12,17 @@ const MyProfileBalanceModal = () => {
             has_close_icon={false}
             is_open={is_modal_open}
             small
-            title={localize('Deriv P2P Balance')}
+            title={localize('Available Deriv P2P Balance')}
         >
             <Modal.Body className='my-profile-balance-modal__body'>
-                <Localize i18n_default_text='Deriv P2P balance = deposits that can’t be reversed' />
+                <div className='my-profile-balance-modal__body-text'>
+                    <Text size='xs'>
+                        <Localize i18n_default_text='Your Deriv P2P balance only includes deposits that can’t be reversed.' />
+                    </Text>
```

Comment:
couldnt add className here as it wasn't applying the styles to it, if I wanted to, i could directly aim for the dc-text and then the name of the class but IMO better we don't directly access it

Reply:
have you tried `<Text as='div' className='my-profile-balance-modal__body-text'`

Reply:
ah that works 👍